user-service/src/main/java/rs/edu/raf/banka1/controller/CapitalController.java

```diff
@@ -187,6 +187,19 @@ public ResponseEntity<Double> estimateBalanceFuture(@PathVariable(name = "future
     public ResponseEntity<Double> estimateBalanceStock(@PathVariable(name = "stockId") Long stockId) {
         return new ResponseEntity<>(capitalService.estimateBalanceStock(stockId), HttpStatus.OK);
     }
+
+    @GetMapping(value = "/balance/options/{optionsId}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Estimate balance for bank account", description = "Estimate balance for bank account")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "Successful operation",
+                    content = {@Content(mediaType = "application/json",
+                            schema = @Schema())}),
+            @ApiResponse(responseCode = "403", description = "Unauthorized"),
+            @ApiResponse(responseCode = "500", description = "Internal server error")
+    })
+    public ResponseEntity<Double> estimateBalanceOptions(@PathVariable(name = "optionsId") Long optionsId) {
+        return new ResponseEntity<>(capitalService.estimateBalanceOptions(optionsId), HttpStatus.OK);
+    }
     /*
```

Comment:
Ovo skoro isto imamo u [BankAccountController.java](https://github.com/RAF-SI-2023/Banka-1-Backend/pull/294/files#diff-c15cdf583528611fd5de534a670ce0b76f73aac8ff86b3d375025c581873a3c5), je l to treba da se ponavlja?

Reply:
Pa stavila sam jer sam videla da ima za stock i future

Reply:
Oke.